utils/easyvpn/cmd/revoke.go

```diff
@@ -17,6 +17,7 @@ func init() {
 	revokeCmd.Flags().BoolVarP(&Commit, "commit", "", true, "git commit changes")
 	revokeCmd.Flags().BoolVarP(&Push, "push", "", true, "git push changes")
 	revokeCmd.Flags().StringVarP(&CertDir, "cert", "c", "cert", "Cert Directory")
+	revokeCmd.Flags().StringVarP(&Network, "network", "", "default", "Network")
```

Comment:
```suggestion
	revokeCmd.Flags().StringVarP(&Network, "network", "n", "default", "Network")
```

Is there a particular reason to not provide the short flag form?

Reply:
Yes, to be coherent with the flag definition already present in sign.go

I can add the short flag on both though.

Reply:
yep, that would help to have both long and short flag  for the 2 commands